src/rules/prefer-in-document.js

```diff
@@ -195,14 +259,20 @@ export const create = (context) => {
         context,
         node.object.object.arguments[0].name
       );
+
+      // Not an RTL query
+      if (!queryNode || queryNode.type !== "CallExpression") {
+        return;
+      }
+
       const matcherNode = node.property;
 
       const matcherArguments = node.parent.arguments;
 
       const expect = node.object.object;
       check({
         negatedMatcher: true,
-        queryNode: (queryNode && queryNode.callee) || queryNode,
+        queryNode: queryNode.callee,
```

Comment:
Because of the `||` operator the coverage was 99.64%. Now it is 100%.
<img width="1181" alt="Screen Shot 2023-07-07 at 20 42 09" src="https://github.com/testing-library/eslint-plugin-jest-dom/assets/24430814/305d6167-af26-48d5-bb07-3548f3af6c75">